pkgs/by-name/ko/komac/package.nix

```diff
@@ -29,34 +32,47 @@ rustPlatform.buildRustPackage {
 
   nativeBuildInputs = [ pkg-config ];
 
-  buildInputs = [
-    dbus
-    openssl
-    zstd
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    darwin.apple_sdk.frameworks.Security
-    darwin.apple_sdk.frameworks.SystemConfiguration
-  ];
+  buildInputs =
+    [
+      dbus
+      openssl
+      zstd
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      darwin.apple_sdk.frameworks.Security
+      darwin.apple_sdk.frameworks.SystemConfiguration
+    ];
 
   env = {
     OPENSSL_NO_VENDOR = true;
     YRX_REGENERATE_MODULES_RS = "no";
     ZSTD_SYS_USE_PKG_CONFIG = true;
   };
 
-  passthru.tests.version = testers.testVersion {
-    inherit version;
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgram = "${placeholder "out"}/bin/komac";
 
-    package = komac;
-    command = "komac --version";
+  passthru = {
+    tests.version = testers.testVersion {
+      inherit version;
+
+      package = komac;
+      command = "komac --version";
+    };
+
+    updateScript = nix-update-script { };
```

Comment:
We got a updating PR as https://github.com/NixOS/nixpkgs/pull/338627.

Why do we need to add this line?

Reply:
r-ryantm checks for repology and github releases without updateScript. It is good to have a updateScript to check for updates directly

Reply:
Ah, thank you for sharing it!

Personally, I would omit adding an empty script until we have applied some tasks for that. But I'm okay with this.

Reply:
You can take a look here: https://ryantm.github.io/nixpkgs/stdenv/stdenv/#var-passthru-updateScript

This script should be enough for us